src/views/verify.tsx

```diff
@@ -61,7 +53,9 @@ export const Verify: React.FunctionComponent = () => {
           try {
             if (rawDocument) {
               setStatus("PENDING");
-              const verify = verificationBuilder(openAttestationVerifiers, { network });
+              const verify = verificationBuilder(openAttestationVerifiers, {
+                provider: new providers.InfuraProvider(network),
+              });
```

Comment:
Would this mean we support the default networks supported by ethers only?

Reply:
um no, if im not wrong u can get these supported [networks](https://docs.ethers.io/v5/api/providers/api-providers/#InfuraProvider).